views/overlays/generic/generic.hbs

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{title}}</title>
+    <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+    <link rel='stylesheet' href='/stylesheets/overlays/generic/generic.css' />
+    <link rel='stylesheet' href='/stylesheets/overlays/characters.css' />
+    <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
+    <script src="/socket.io/socket.io.js"></script>
+    <script src="/javascripts/remote/obs-remote.min.js"></script>
+  </head>
+  <body>
+    <div id="background" class="container"></div>
+    <div id="main-container" class="container">
+      <div class="title playerText"></div>
+      <div class="tourneyInfo playerText"></div>
+      <div class="commentators playerText"></div>
+      <div class="lPort"></div>
+      <div class="rPort"></div>
+      <div class="lScore playerText"></div>
+      <div class="lPlayer playerText"></div>
+      <div class="lIcons">
+        <img class="lCharacter" src="http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif">
+        <p class="lState"></p>
+      </div>
+      <div class="rIcons">
+        <img class="rCharacter" src="http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif">
+        <p class="rState"></p>
+      </div>
+      <div class="rPlayer playerText"></div>
+      <div class="rScore playerText"></div>
+    </div>
+    <script>
+    OBSRemote.prototype._authHash = OBSRemote.prototype._webCryptoHash;
```

Comment:
Could you pull all this scripting out into a JS file? I'd like to move away from having it all inline.